source/conf.py

```diff
@@ -0,0 +1,180 @@
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+import sphinx_bootstrap_theme
+
+# -- Project information -----------------------------------------------------
+
+project = 'Mantid Project'
+copyright = '2021, Mantid'
+author = 'Mantid'
+
+# The short X.Y version
+version = ''
+# The full version, including alpha/beta/rc tags
+release = '6.1'
+
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = None
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'bootstrap'
+html_theme_path = sphinx_bootstrap_theme.get_html_theme_path()
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+html_theme_options = {
+    'navbar_title': " ",  # Doesn't accept empty string
+    'navbar_class': "navbar-right", 
+    'navbar_links': [
+        ("Downloads", "https://download.mantidproject.org/", 1),
+        ("Guides", "Documentation"),
+        ("Docs", "https://developer.mantidproject.org/", 1),
+        ("News", "https://forum.mantidproject.org/c/news/8", 1),
+        ("Forums", "https://forum.mantidproject.org/", 1),
+        ("Contact Us", "Contact")
+    ],
+    'navbar_pagenav': False,
+    'source_link_position': None,
+}
+
+html_static_path = ["_static"]
+
+html_logo = "_static/images/Mantid_Logo_Transparent.png"
+
+# -- Options for HTMLHelp output ---------------------------------------------
```

Comment:
I don't imagine we will use any of the other builders for this. To try and keep this file simple can we remove the following sections until the `def setup`?